Add index.html, styles.css / Update readme.md

JuanDiegoGuerra opened the pull request:
Added the main 2 files of this project and also updated the README.
It was a very hard working project for me, even that I know it was only the "setup" for the real project 😅.
I'll be patiently wait for a review and feedback 😄.

JuanDiegoGuerra committed:
Create index.html

Created the main page from the site with the structure needed to show it as a portfolio.

JuanDiegoGuerra committed:
Create styles.css

Creating the css file to stylish the main page.

JuanDiegoGuerra committed:
Add 10 Images

Adding 10 images the project will use.
x1 Page Logo (my esports org)
x7 Figures for background
x1 Complete background img
x1 Social Logo (github)

JuanDiegoGuerra committed:
Update README.md

Updating some sections of the README file.

JuanDiegoGuerra committed:
Update styles.css

Deleting a figure that was causing visual problems for the lectors in the phone size.

JuanDiegoGuerra committed:
Update index.html

Update index to fix one linter problem.

JuanDiegoGuerra committed:
Update styles.css

Updated the css file to fix all stylelint erros.

IngDavidTM commented on css/styles.css:
```diff
@@ -0,0 +1,215 @@
+  .primary-logo {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 505px) {
+  .index-page {
+    background: white;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .menu {
+    display: none;
+  }
+
+  .flex-nav-list {
+    display: block;
+    list-style-type: none;
+    width: 100%;
+    text-align: center;
+    padding-top: 10px;
+    margin-left: -20px;
+  }
+
+  .flex-nav-list .list-item {
+    display: block;
+    margin-right: -100px;
+    border-top: 1px solid rgb(0, 0, 0);
+  }
+
+  .primary-logo {
+    width: fit-content;
+    margin-left: 10px;
+  }
+
+  #headline-section {
+    margin-top: 30px;
+  }
+}
```
- [ ] Although you have done an excellent job designing the mobile version of your project :+1:, it is important to note that as a project requirement, it should be mobile-first. This means that you should avoid using media queries to adapt the design to different screen sizes after the fact, and instead focus on designing for mobile screens as the primary consideration. The reason for this is that it can lead to a more streamlined and efficient design process, as well as improved user experience on mobile devices :+1:.
- [ ] In addition, it is mandatory that the minimum screen size for mobile devices should be 375px, so please ensure that there are no design flaws at that size, as illustrated in the video below (nav bar errors).


https://user-images.githubusercontent.com/105117832/227159380-f1169b02-3ab6-461c-ab98-2d97baa6c8d6.mp4

**Required design**

![Screenshot from 2023-03-23 04-26-19](https://user-images.githubusercontent.com/105117832/227159794-fda4bd49-7621-42c1-8d58-3302d951e862.png)

IngDavidTM commented on index.html:
```diff
@@ -0,0 +1,64 @@
+        </nav>
+        </header>
+        <section id="headline-section">
+            <h4 class="headline-title"> Hi there!👋🏼
+                I'm JD 🤭</h4>
```
- [ ] Well done on correctly identifying the title of your project :+1:. However, to ensure semantic accuracy, it is recommended that the main title of your page should be marked up with an `h1` tag instead of an `h4` tag. This is because the `h1` tag carries more weight in terms of search engine optimization and assists in organizing the structure of your page for both users and web crawlers. Using an `h1` tag for the main title will help to improve the accessibility and usability of your webpage :+1: .

IngDavidTM commented on css/styles.css:
```diff
@@ -0,0 +1,215 @@
+  margin-bottom: 5px;
+}
+
+#headline-section {
+  height: 916px;
+  margin-left: 18%;
+  margin-right: 18%;
+  margin-top: 150px;
+  background-image: none;
+}
```
- [ ] Although you did an excellent job :+1:, please note that as a project requirement, the section needs to be aligned using flexbox. This means that you should use flexbox properties and values to align the elements within the section. The reason for this is that flexbox provides a powerful and flexible way to organize and align content within a container, making it easier to create responsive layouts and adjust to different screen sizes :+1:.
- [ ] Well done on the styling of your project :+1:. However, it is necessary that your portfolio closely resembles the Figma template as per the project requirements. Kindly be aware that the mobile version has a distinct background image that needs to be displayed from 375px. Additionally, please note that the icons are not the same. To address this, you may use different tools, and I suggest using [Font Awesome](https://fontawesome.com/icons).
- [ ] Also there are other differences listed below:
  - The title should have two different colors :+1:.
  - The size of `p` should be 20px :+1:
  - You should add the button  with the chevron down in the bottom of your page :+1:
  - Kindly check the padding and margins of your portfolio :+1:

**Your design**

![Screenshot from 2023-03-23 04-46-30](https://user-images.githubusercontent.com/105117832/227165022-37c70862-39db-4f6b-b130-afd00ef674e5.png)

**Figma**

![Screenshot from 2023-03-23 04-46-52](https://user-images.githubusercontent.com/105117832/227165111-202cc798-150a-4f72-a452-3e9b4c1ec2e2.png)

IngDavidTM reviewed (changes requested):
## Hi @JuanDiegoGuerra  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JuanDiegoGuerra committed:
Update index.html

Fixed semantic accuracy.
Added the correct icons to the footer.

JuanDiegoGuerra committed:
Create mobile_first.css

Created a new css to show main site to be mobile-first.
Added some imgs too in order to fix background issues
PD: Not deleting the other css file because will be helpfull for the next project.

JuanDiegoGuerra commented:
### @IngDavidTM **Hello!**
I just finished everything 🤩. It was kinda hard working for me. It wasn't hard at all but I think there was a lot of information I had to review in order to complete it. Hope I fixed everything you tell me to. I'll be waiting for your feedback. Thanks again!

👉🏼**Mobile-First** design fixed and looking like the figma template I used: ✅

![SetupPortfolioAndMobileProjectFirstChanges](https://user-images.githubusercontent.com/45726631/227277940-d2334bcc-8141-4f69-88cd-ef8189cff00d.JPG)

TadesseDev commented on index.html:
```diff
@@ -29,31 +29,18 @@
         <section id="headline-section">
-            <h4 class="headline-title"> Hi there!👋🏼
-                I'm JD 🤭</h4>
+            <h1 class="headline-title"> Hi there!👋🏼 <br>
+                I'm JD.</h1>
+                <h2 class="headline-sub-title"> I'm a Micronout 👨🏼‍🚀💻</h2>
```
- [Optional] Dear @JuanDiegoGuerra, please consider removing icons that are not part of the figma design (👋🏼👨🏼‍🚀💻). As front-end developers, we need to mimic the design and deliver something identical.

JuanDiegoGuerra replied:
Hello @TadesseDev, I just removed the icons. I know this is part of being professional but I think that sometimes your work should have some of your **spirit** or **personality**, cause that won't make you see less professional or serious and that also differences us from the computers hehe. I think this has a value for me and in some point you could remember the guy who tell you a joke in a job interview insteed the others that only went to do what they had to do. I really appreciate your feedback and I'm continue fixing the issues you pointed.

TadesseDev replied:
I know right😊. But the thing is. we are considering ourselves as someone who have just received a template to work with. The test is all about delivering the giving template. Once you have completed the portfolio project you will have the freedom of customizing it to the point you like it anyhow. I'm sorry for giving you a hard time brother @JuanDiegoGuerra  😊. But I'm sure you are going to learn a lot as we all did.

JuanDiegoGuerra replied:
No problems mate @TadesseDev. I'm glad I'm reiciving good and helpful feedback! 💪🏼

TadesseDev commented on index.html:
```diff
@@ -0,0 +1,51 @@
+                I want to learn everything I need to be prepared for a programmer's job. You can contact me:
+            </p>
+               <footer>
+                <div id="contact">
+                    <ul>
+                        <li class="icon">
+                            <a href="https://github.com/JuanDiegoGuerra" rel="noopener" target="_blank">
+                                <img src="img/social media icons.png" alt="github link">
+                            </a></li>
+                    </ul>
+                </div>
```
- [ ] Good job adding the `down arrow icon` as suggested by the previous reviewer. But please notice you need to make sure it's added to the right place on the webpage. currently, it's overlapping with your social media icons. This creates a bad experience for the end user, Please consider aligning it to the bottom and by its own. 

### Your design on `375px` screen size
![image](https://user-images.githubusercontent.com/69077061/227435251-6efe69ce-9b22-4183-b78a-724b44f5afba.png)

TadesseDev commented on css/mobile_first.css:
```diff
@@ -0,0 +1,147 @@
+  top: 25px;
+}
+
+.index-page {
+  background-image:
+    url("../img/header-illsutration-mobile.png"),
+    url("../img/Action.png");
+  background-repeat: no-repeat;
```
- [ ] Dear @JuanDiegoGuerra, You did great adding the background image as suggested. I also notice the previous reviewer mention to look out for your margin and padding. As you can see your background image does not take the whole width of the page compared with the figma design. the background should take 100% of the view width.

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/227435721-4d411299-c97c-4f56-a731-b51f361e0d3d.png)
### Your design on 375px screen size
![image](https://user-images.githubusercontent.com/69077061/227435848-f61ec5e8-dc63-4819-ab97-8ae093bfac35.png)

JuanDiegoGuerra replied:
Hello again @TadesseDev . I don't know if this was a problem of my "view perspective" because I was also working with that pixel size (375) and also tested with some different phone sizes, like some Samgsung and Google devices, and they were working fine, both the background and the "action" down-arrow icon. I only had that problem with the Iphone screen size. Maybe is not optimized for all the devices but I still re-fixed it to make it good for the most common phone size and also for the specific sizes that was requierd for the project. But maybe I'm wrong, idk. I will wait patienly for another review, hope it can be you again. Thank you for your time!
I'm attaching some images that shows different size tests:

- Most common phone size: ✅
![ResponsibeMobileSize-most_common_size](https://user-images.githubusercontent.com/45726631/227446685-70421432-f9ab-4c53-bdd2-c838e3d4e4c1.JPG)

- Samsung Galaxy S8 size: ✅
![ResponsibeMobileSize-samsung_galaxy_s8](https://user-images.githubusercontent.com/45726631/227446772-8843ee7f-b314-4b17-9e0d-7955a07ec228.JPG)

- Iphone SE size: ✅
![ResponsibeMobileSize-Iphone_SE](https://user-images.githubusercontent.com/45726631/227446870-42875efb-52fd-4652-92f3-55b74976f90c.JPG)

TadesseDev reviewed (changes requested):
### Hi @JuanDiegoGuerra 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7](https://user-images.githubusercontent.com/69077061/212385281-4797c58a-d358-423a-aa7c-832c22203865.gif)




## Highlights
 - [ ] Good job addressing most of the changes requested by the previous reviewer ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JuanDiegoGuerra committed:
Update index.html / mobile_first.css

Fixed the issues pointed on the last review.
Sumary: deleted icons, fixed bg and down-arrow size to be optimize for most common phone sizes,

amejid reviewed (approved):
Hi @JuanDiegoGuerra, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/YKdm91XONLTaw/giphy-downsized-medium.gif"/>

## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
